packages/components/psammead-promo-separator/README.md

```diff
@@ -0,0 +1,50 @@
+# psammead-promo-separator - [![Known Vulnerabilities](https://snyk.io/test/github/bbc/psammead/badge.svg?targetFile=packages%2Fcomponents%2Fpsammead-promo-separator%2Fpackage.json)](https://snyk.io/test/github/bbc/psammead?targetFile=packages%2Fcomponents%2Fpsammead-promo-separator%2Fpackage.json) [![Storybook](https://raw.githubusercontent.com/storybooks/promo-separator/master/badge/badge-storybook.svg?sanitize=true)](https://bbc.github.io/psammead/?path=/story/promo-separator--default) [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/bbc/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/psammead-promo-separator.svg)](https://www.npmjs.com/package/@bbc/psammead-promo-separator) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/bbc/psammead/blob/latest/CONTRIBUTING.md)
+
+## Description
+
+The `@bbc/psammead-promo-separator` package exports a single separator line component.
+It uses a `hr` HTML element.
```

Comment:
Should it be an `hr`?

> The HTML `<hr>` element represents a thematic break between **paragraph-level elements**
> Historically, this has been presented as a horizontal rule or line. While it may still be displayed as a horizontal rule in visual browsers, this element is now defined in semantic terms, rather than presentational terms, so if you wish to draw a horizontal line, you should do so using appropriate CSS.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/hr

And the examples in https://html.spec.whatwg.org/multipage/grouping-content.html#the-hr-element

These read to me as `hr`s should only be used in article pages, and this separator should just be a styled `div`